tensorflow_cloud/containerize.py

```diff
@@ -228,6 +242,13 @@ def _generate_name(self):
         unique_tag = str(uuid.uuid4()).replace("-", "_")
         return "{}/{}:{}".format(self.docker_registry, "tf_cloud_train", unique_tag)
 
+    def _base_image_exist(self):
+        """Check whether the docker base image exist on dockerhub. 
+        use docker api v2 to check if base image is available.
+        """
+        repo_name, tag_name = self.docker_base_image.split(":")
+        r = requests.get("http://hub.docker.com/v2/repositories/{}/tags/{}".format(repo_name, tag_name))
```

Comment:
Please format files using `black`. pip install black, black filename.py.

Reply:
Thanks for the information.  Formatted.